_includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-product-offer-feature.md

```diff
@@ -0,0 +1,1770 @@
+| ProductOfferStorageCriteria                | class    | created | src/Generated/Shared/Transfer/ProductOfferStorageCriteriaTransfer    |
+| ProductOfferStore                          | class    | created | src/Generated/Shared/Transfer/ProductOfferStoreTransfer              |
+| ProductOfferValidity                       | class    | created | src/Generated/Shared/Transfer/ProductOfferValidityTransfer           |
+| ProductOfferValidityCollection             | class    | created | src/Generated/Shared/Transfer/ProductOfferValidityCollectionTransfer |
+| EventEntity                                | class    | created | src/Generated/Shared/Transfer/EventEntityTransfer                    |
+| ProductConcretePageSearch                  | class    | created | src/Generated/Shared/Transfer/ProductConcretePageSearchTransfer      |
+| ProductPageSearch.merchantReferences       | property | created | src/Generated/Shared/Transfer/ProductPageSearchTransfer              |
+| ProductPayload.merchantReferences          | property | created | src/Generated/Shared/Transfer/ProductPayloadTransfer                 |
+| ProductStorageCriteria.merchantReference   | property | created | src/Generated/Shared/Transfer/ProductStorageCriteriaTransfer         |
+| ProductView.productOfferReference          | property | created | src/Generated/Shared/Transfer/ProductViewTransfer                    |
+| Merchant.merchantReference                 | property | created | src/Generated/Shared/Transfer/MerchantTransfer                       |
+| Merchant.status                            | property | created | src/Generated/Shared/Transfer/MerchantTransfer                       |
+
+{% endinfo_block %}
+
+### 3) Add translations
+
+Generate a new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+### 4) Configure export to Redis and Elasticsearch
+
+To configure export to Redis and Elasticsearch, take the following steps:
+
+#### Set up publisher plugins:
+
+| PLUGIN                                                  | SPECIFICATION                                                                                                                                 | PREREQUISITES | NAMESPACE                                                                                          |
+|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------|
+| ProductConcreteProductOffersDeletePublisherPlugin       | Finds and deletes product concrete offer storage entities by the given concreteSkus.                                                          |               | Spryker\Zed\ProductOfferStorage\Communication\Plugin\Publisher\ProductConcreteOffers               |
```

Comment:
We have to add the following missed plugins:
* ProductConcreteProductOffersStoreDeletePublisherPlugin
* ProductConcreteProductOffersStoreWritePublisherPlugin
* ProductOfferStoreDeletePublisherPlugin
* ProductOfferStoreWritePublisherPlugin
* ProductOfferPublisherTriggerPlugin
* MerchantProductOfferProductConcreteExpanderPlugin
* DefaultProductOfferStorageCollectionSorterPlugin